torch/utils/_sympy/value_ranges.py

```diff
@@ -564,7 +564,7 @@ def sqrt(x):
         x = ValueRanges.wrap(x)
         if x.lower < 0:
             return ValueRanges.unknown()
-        return ValueRanges.increasing_map(x, sympy.sqrt)
+        return ValueRanges.increasing_map(x, OpaqueUnaryFn_sqrt)
```

Comment:
I mean, sure, but otherwise just use `math.sqrt` for this to be a bit less magical.

Reply:
Actually it's necessary to use the sympy function, esp in the latest version of my PR, because I need to handle infinity/negative infinity.